presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
+            throws Exception
+    {
+        try {
+            TypedSet typedSet = new TypedSet(BIGINT, 10, FUNCTION_NAME);
+
+            for (int i = 0; i <= TypedSet.FOUR_MEGABYTES + 1; i++) {
+                Block block = createLongsBlock(Collections.nCopies((int) 1, (long) i));
+
+                typedSet.add(block, 0);
+            }
+
+            fail("expected exception");
```

Comment:
"In Presto, we only have empty lines for readability purpose" i came up with my rule because "readability" is highly subjective as I've worked in many places. when i had the autonomy, i just used my rules to not think.

Going forward, just show me your general rules for when you prefer blank lines, and i'll adapt just fine. I'll look for whatever patterns you guys use and infer rules as well.